spec/fixtures/files/metadata_dynamics.xml

```diff
@@ -339,6 +339,78 @@
         <NavigationProperty Name="contact_activity_parties" Type="Collection(mscrm.activityparty)" Partner="partyid_contact"/>
         <NavigationProperty Name="Contact_Letters" Type="Collection(mscrm.letter)" Partner="regardingobjectid_contact_letter"/>
       </EntityType>
+      <EntityType Name="role" BaseType="mscrm.crmbaseentity">
```

Comment:
I am not seeing the Alias Attribute in the test. am I missing something?

Reply:
The namespace and alias are specified on line 12 of this file